tikv-client/src/test/java/com/pingcap/tikv/meta/MetaUtils.java

```diff
@@ -119,7 +119,9 @@ public TiTableInfo build() {
         name = "Table" + tid;
       }
       return new TiTableInfo(
-          tid, CIStr.newCIStr(name), "", "", pkHandle, columns, indices, "", 0, 0, 0, 0);
+          tid, CIStr.newCIStr(name), "", "", pkHandle, columns,
+          indices, "", 0, 0, 0
+          , 0, null);
```

Comment:
What does setting partitionInfo to null mean exactly?

Reply:
It means the table is a normal table rather than a partition table.